common/io/CMakeLists.txt

```diff
@@ -16,17 +16,19 @@ add_library(${PROJECT_NAME} ${cga_library_type}
         src/hts_fasta_parser.cpp)
 target_link_libraries(${PROJECT_NAME} PUBLIC hts)
 
+include_directories(../utils/include)
```

Comment:
If it turns out we cannot replace `claragenomics/utils/filesystem.hpp` with `std::filesystem`:

`claragenomics/utils/filesystem.hpp` should be added to `utils` library (or turned into a separate library) and included properly